webrtc.html

```diff
@@ -7484,14 +7485,15 @@ <h2>
         possible designs have been discussed in <a href=
         "https://github.com/w3c/webrtc-pc/issues/1283">GitHub issue 1283</a>.
       </p>
-      <p>
-        When video is rescaled, for example for certain combinations of width
-        or height and {{RTCRtpEncodingParameters/ scaleResolutionDownBy}}
-        values, situations when the resulting width or height is not an integer
-        may occur. In such situations the user agent MUST use <a href=
-        "https://tc39.github.io/ecma262/#eqn-floor">the integer part of the
-        result</a>. What to transmit if the integer part of the scaled width or
-        height is zero is implementation-specific.
+      <p id="rtp-video-rescaled">
+        When video is rescaled using {{RTCRtpEncodingParameters/scaleResolutionDownBy}},
```

Comment:
Rephrase to clarify rescaling requirement applies over time.
```suggestion
        Whenever video is rescaled as a result of {{RTCRtpEncodingParameters/scaleResolutionDownBy}},
```